app/gui/view/graph-editor/src/builtin/visualization/java_script/table.js

```diff
@@ -1,4 +1,8 @@
 /** Table visualization. */
+loadScript('https://cdn.jsdelivr.net/npm/ag-grid-community/dist/ag-grid-community.min.js')
+// loadScript('https://cdn.jsdelivr.net/npm/ag-grid-enterprise@29.1.0/dist/ag-grid-enterprise.min.js')
+loadStyle('https://cdn.jsdelivr.net/npm/ag-grid-community/styles/ag-grid.css')
+loadStyle('https://cdn.jsdelivr.net/npm/ag-grid-community/styles/ag-theme-alpine.css')
```

Comment:
Just a note here. Nothing to fix here, as we are doing it wrongly in other places as well, but basically this is bad, as it makes the app not working without internet connection. We need to fix this and other similar places one day. CC @sylwiabr